integration-tests/tests/advanced-happy-path.spec.ts

```diff
@@ -148,7 +148,7 @@ describe('Advance Happy path', () => {
           Common.checkResponseBodyAndStatusCode(
             route,
             componentInfo.get('deploymentBodyUpdated'),
-            30000,
+            40000,
```

Comment:
can we get variable maximum retries, rather than longer wait interval?

Reply:
Added maxRetryNum